client/frame.js

```diff
@@ -223,6 +223,16 @@
   }
 
   function frameListener(event) {
+    // is this message for us?
+    // events from iframes don't have an opener
+    // ensure that the iframe is one of ours
+    if (event.source.opener ||
+        !$("iframe").filter((i,el) => el.contentWindow === event.source).parents(".item").hasClass("frame")) { 
```

Comment:
A reduced way to ensure the iframe is inside a frame plugin item: 
```suggestion
        !$(".item.frame iframe").filter((i,el) => el.contentWindow === event.source)) { 
```

Reply:
Yes, also I'd expect that to be more efficient.